vue-frontend/src/components/home-new/ContributionSection.vue

```diff
@@ -1,27 +1,30 @@
 <template>
-  <div class="tw-relative">
-    <div class="tw-container tw-mt-2 md:tw-mt-4">
-      <div
-        class="tw-sticky tw-w-full tw-h-[30vh] sm:tw-h-[25vh] lg:tw-h-[35vh] tw-z-[1] tw-top-[6rem] tw-bg-white tw-pt-12 tw-text-center"
+  <div
+    class="tw-container lg:tw-flex lg:tw-flex-row-reverse tw-justify-center tw-align-center tw-max-w-[1878px] lg:tw-pt-20 tw-text-center lg:tw-text-left"
+  >
+    <div
+      class="lg:tw-basis-[35%] 2xl:tw-basis-[25%] tw-my-auto tw-mx-auto lg:tw-ml-11 2xl:tw-mx-0 tw-w-[290px] sm:tw-w-full tw-pb-14 md:tw-pb-20 lg:tw-pb-0"
```

Comment:
Why its `290px`? Let's ignore manual points for horizontal spaces, I think we can achieve this by flex properties

Reply:
There was issue of underline that's why given fixed width so that it never break.